### ISSUE TITLE
0.135.0 version of emulator crashes with unsupported architecture aarch64

### ISSUE THREAD

**Issue by ce07c3** [googleapis/java-bigtable#1525]
The 0.135.0 jar pulled here:

```xml
<dependency>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-bigtable-emulator</artifactId>
     <version>0.135.0</version>
     <scope>test</scope>
</dependency>
```

Crashes with an `aarch64` exception. The `.class` file shows that the conditional does not check for `arm`.

Where can I get a maven dep that has the newest version of the code, the one on main in this repository?

**Comment by kolea2** [comment]
Hi @ce07c3, I believe the feature you're looking for was included in emulator version 0.142.0 (see https://github.com/googleapis/java-bigtable/releases/tag/v2.5.0).

Currently, the latest version of the emulator is 0.153.0 -  https://search.maven.org/artifact/com.google.cloud/google-cloud-bigtable-emulator

Please let us know if you have any other issues. Thanks!

**Comment by ce07c3** [comment]
> Hi @ce07c3, I believe the feature you're looking for was included in emulator version 0.142.0 (see https://github.com/googleapis/java-bigtable/releases/tag/v2.5.0).
> 
> Currently, the latest version of the emulator is 0.153.0 - https://search.maven.org/artifact/com.google.cloud/google-cloud-bigtable-emulator
> 
> Please let us know if you have any other issues. Thanks!

Thank you so much for your swift reply. I will try that out - I must have missed that somehow. I will close this ticket for now. 🙂
